tools/releaser/scripts/preview-version.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ueo pipefail
+
+script_path=$(dirname "$0")
+# shellcheck source=/dev/null
+source "$script_path/extract-version.sh"
+# shellcheck source=/dev/null
+source "$script_path/version-paths.sh"
+
+OLD_PACKAGE="go.mongodb.org/atlas-sdk/${SDK_MAJOR_VERSION}"
+NEW_PACKAGE="github.com/mongodb/atlas-sdk-go"
+examples_path="$script_path/../../../examples"
+
+echo "Delete specific version $SDK_MAJOR_VERSION from examples go.mod"
+(cd "$examples_path" && go mod edit -droprequire="$OLD_PACKAGE")
```

Comment:
because the replace,  if not done this is the change:
![change](https://github.com/user-attachments/assets/8b754a61-09d3-4018-8809-2b7d3d9fde95)

and we get this error when running in examples folder go test ./...

```
go: errors parsing go.mod:
go.mod:12:2: require github.com/mongodb/atlas-sdk-go: version "v20241023002.1.0" invalid: should be v0 or v1, not v20241023002
```

Reply:
Yes. That is what I have expected. As alternative you can discuss with team if we need to run tests for examples or even compile them on the dev version as this will lead to many false compilation issues where examples outdated.
There are pros and cons of this approach thus likely not good to make this change in this PR.

Reply:
if we don't do this code-health PR check will fail, I think it's good that the generated PR has all PR checks green